.circleci/config.yml

```diff
@@ -73,21 +95,45 @@ commands:
                 name: Save Yarn Package Cache
                 paths:
                     - ~/.cache/yarn
+    restore_yarn_tools_cache:
+        description: Restores and saves the node_modules directories of the tools package.
+        steps:
+            - restore_cache:
+                key: 2-yarn-{{ arch }}-{{ checksum "tools/yarn.lock" }}-{{ checksum "tools/package.json" }}
+                name: Restore Yarn Tools Package Cache
+            - run:
+                command: yarn --frozen-lockfile --cache-folder ~/.cache/yarn-tools
+                name: Yarn tools
+                working_directory: tools
+            - save_cache:
+                key: 2-yarn-{{ arch }}-{{ checksum "tools/yarn.lock" }}-{{ checksum "tools/package.json" }}
+                name: Save Yarn Tools Package Cache
+                paths:
+                    - ~/.cache/yarn-tools
+                    - tools/node_modules
 jobs:
     build_android:
         docker:
             - image: circleci/android:api-29-node
         environment:
             FASTLANE_SKIP_UPDATE_CHECK: true
-        resource_class: medium+
```

Comment:
Are you sure this is enough?
After doing a lot of testing I'm very sure that this can not be enough for a react native project

Reply:
not enough for what? Pipeline is not failing currently. So it should be fine? But i will upgrade it, as soon as it is possible. Lunes is still in the free plan, there we can't use medium+

Reply:
Actually, it is failing :see_no_evil: E.g. here: https://app.circleci.com/pipelines/github/digitalfabrik/lunes-app/408/workflows/3bf88902-46ed-44b6-95f0-34cf06d91304/jobs/686

Reply:
Changing it will not help, as Lunes is in the free plan and medium+ is only available in performance plan.

Reply:
@klinzo The example you linked, "uses" medium+ class, but is downgraded due to free plan.

Reply:
Oke Lunes is now in Performance Plan and medium+ class is specified and used! :tada: